client/src/components/Recipe/RecipeIngredients.js

```diff
@@ -26,7 +25,7 @@ const RecipeIngredients = ({ Ingredients, id }) => {
 
 RecipeIngredients.propTypes = {
   Ingredients: PropTypes.array,
-  id: PropTypes.number,
+  // id: PropTypes.any,
```

Comment:
Do you still need this? otherwise, you could remove it ;)

Reply:
I'll check that, thanks!:)